src/kernel-install/50-depmod.install

```diff
@@ -2,7 +2,14 @@
 # -*- mode: shell-script; indent-tabs-mode: nil; sh-basic-offset: 4; -*-
 # ex: ts=8 sw=4 sts=4 et filetype=sh
 
-[[ $1 == "add" ]] || exit 0
 [[ $2 ]] || exit 1
 
-exec depmod -a "$2"
+if [[ $1 == "add" ]]; then
+    exec depmod -a "$2"
+fi
+
+if [[ $1 == "remove" ]]; then
+    exec rm -f /lib/modules/$2/modules.{alias{,.bin},builtin.bin,dep{,.bin},devname,softdep,symbols{,.bin}}
+fi
```

Comment:
I think it'd be more natural to use a switch statement:
```
case "$1" in
add) ...;;
remove) ...;;
*) exit 1;;
esac
```
but that's a minor thing.

The bigger issue is this hardcoded list of stuff to remove. Can't we just nuke `/lib/modules/$2/modules.*`?